.github/workflows/main.yml

```diff
@@ -0,0 +1,116 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+env:
+  # needed by coveralls
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CIBW_BUILD: "cp36-* cp37-* cp38-*"
+  CIBW_BEFORE_BUILD: "pip install numpy==1.17.3 cython>=0.29.14 setuptools"
+  CIBW_TEST_REQUIRES: "pytest"
+  CIBW_TEST_COMMAND: "pytest -v {project}/tests"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: wheels on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    # only run on internal push's or external PRs, not on internal PRs
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions\setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install cibuildwheel>=1.7.0
+      - name: Build wheels
+        run: |
+          python -m cibuildwheel --output-dir dist
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./dist/*.whl
+
+  build_sdist:
```

Comment:
Why running sdist on all os and py versions? I think py38 on linux should be enough.

Reply:
Testing OpenMP on all combinations was my intention...

Reply:
Oh... clever ;-)

Reply:
But then: why not py39?

Reply:
... a little bit embarrassing...I missed the release of py39... :blush:
I'll let the current actions jobs run through and then add py39